README.md

```diff
@@ -96,13 +96,13 @@ For example, download TIMED.h5 . Make sure to place it in this directory.
 
 
 ```
-python3 predict.py --path_to_dataset $DATASET_PATH --path_to_model MODEL_PATH
+python3 predict.py --path_to_dataset $DATASET_PATH --path_to_model MODEL_PATH --output_dir OUTPUT_DIRECTORY
 ```
 
 eg.
 
 ```
-python3 predict.py --path_to_dataset data.hdf5 --path_to_model TIMED.h5
+python3 predict.py --path_to_dataset data.hdf5 --path_to_model TIMED.h5 --output_dir test_outputs
 ```
```

Comment:
Here I'd add a separate example for the output directory. Mainly because if a user is copying and pasting commands they would need to go to the test_outputs folder to see the output which may not be entirely obvious. 

I'd suggest to modify it to: 

```
python3 predict.py --path_to_dataset data.hdf5 --path_to_model TIMED.h5 --output_dir .
```